bin/release.js

```diff
@@ -196,7 +197,7 @@ const checkReleaseStatus = coroutine(function * () {
   let tags
 
   try {
-    tags = yield taggedVersions.getList()
+    tags = yield taggedVersions.getList({rev: 'HEAD'})
```

Comment:
Just realised that if a repo with v1.1.0, v1.1.1 and v2.0.0. If v2.0.0 branched off v1.1.0, `taggedVersions.getList({rev: 'HEAD'})` will resolve to `[Tag<v2.0.0>, Tag<1.1.0>]`; because v1.1.0 is not an ancestor of v2.0.0.

I will fix it so that it query the next tag and the previous tag in separate operation.